rangeseekbar/src/main/java/org/florescu/android/rangeseekbar/RangeSeekBar.java

```diff
@@ -116,6 +117,7 @@
     private int textSize;
     private int distanceToTop;
     private RectF rect;
+    private RectF rectActive;
```

Comment:
I'm thinking that this doesn't need another rect on top of it, could be done with the one we have that's already drawing the line, just need to adapt its `top` and `bottom` when we draw it if it's active